doc/nrf/ug_thingy53.rst

```diff
@@ -106,57 +105,46 @@ Note that building for the non-secure application core is not fully supported.
 | nRF5340 SoC - Network core     |``thingy53_nrf5340_cpunet``                               |
 +--------------------------------+----------------------------------------------------------+
 
+.. note::
+   The |NCS| samples and applications that are compatible with Thingy:53 follow the :ref:`thingy53_app_guide`.
+   This is needed, because the samples must, among others, use consistent partition map and bootloader configuration.
+
 |NCS| by default uses :ref:`ug_multi_image` for Thingy:53.
-Because of this, when you choose :file:`thingy53_nrf5340_cpuapp` when building a sample or application that is fully compatible with Thingy:53, you will generate firmware for both the application core and network core:
+Because of this, if you choose ``thingy53_nrf5340_cpuapp`` when building a sample or application that is compatible with Thingy:53, you will generate firmware for both the application core and network core:
```

Comment:
```suggestion
Because of this, when you choose ``thingy53_nrf5340_cpuapp`` when building a sample or application that is compatible with Thingy:53, you will generate firmware for both the application core and network core:
```
We are instructing them to use it. Not wondering if they might end up using it.